src/components/ec-table/ec-table.vue

```diff
@@ -14,6 +14,7 @@
         class="ec-table"
       >
         <ec-table-head
+          v-if="!canShowCustomSlot"
```

Comment:
not sure we should link displaying of the header and slot in the table body this way.

Let's add a prop `isTableHeaderHidden` it also could have three values and by default header is hidden when slot is shown.